Sources/MapboxMaps/Annotations/ViewAnnotation.swift

```diff
@@ -130,17 +148,17 @@ public final class ViewAnnotationManager {
     ///
     /// - Returns: `ViewAnnotationOptions` if view was found and `nil` otherwise.
     public func options(forFeatureId identifier: String) -> ViewAnnotationOptions? {
-        return viewAnnotation(forFeatureId: identifier).flatMap(options(for:))
+        return view(forFeatureId: identifier).flatMap(options(for:))
```

Comment:
I think we could come up with a more efficient implementation. This one results in an extra call to `options(forViewAnnotationWithId:)` that could be avoided.